pkgs/tools/graphics/gif2apng/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, fetchzip
+, fetchpatch
+, zlib
+, zopfli
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gif2apng";
+  version = "1.9";
+
+  src = fetchzip {
+    url = "mirror://sourceforge/gif2apng/gif2apng-${version}-src.zip";
+    stripRoot = false;
+    hash = "sha256-rt1Vp4hjeFAVWJOU04BdU2YvBwECe9Q1c7EpNpIN+uE=";
+  };
+
+  patches = [
+    (fetchpatch {
+      url = "https://sources.debian.org/data/main/g/gif2apng/1.9%2Bsrconly-3%2Bdeb11u1/debian/patches/10-7z.patch";
```

Comment:
I think those links are not save to use because they disappear when new versions are released

Reply:
It should be fine, I see all versions at https://sources.debian.org/data/main/g/gif2apng/

Reply:
Those are only the versions that are in some current release series. See https://github.com/NixOS/nixpkgs/commit/fb7c2cdb519dcffb3cd3b29b8179a306afaecd74 and https://github.com/NixOS/nixpkgs/blob/2341c80301d63eecb06f41b69b3cd1bee2f243b9/pkgs/development/libraries/libconfuse/default.nix#L25